e2e/jobs/cgroup.hcl

```diff
@@ -26,7 +26,7 @@ job "cgroup" {
       config {
         command = "/usr/bin/cat"
         args    = ["/proc/self/cgroup"]
-        unveil  = ["r:/proc/self/cgroup"]
+        unveil  = ["r:/proc"]
```

Comment:
`/proc/self` is the shim process; now that we fork+exec instead of just exec, `cat` is a child and cannot see _it's_ `/proc/self`, which will not have been made available by landlock. There's Tylenol in the kitchen.